apps/web/pages/event-types/index.tsx

```diff
@@ -675,7 +679,11 @@ const EventTypeListHeading = ({
       <Avatar
         alt={profile?.name || ""}
         href={teamId ? `/settings/teams/${teamId}/profile` : "/settings/my-account/profile"}
-        imageSrc={`${WEBAPP_URL}/${profile.slug}/avatar.png` || undefined}
+        imageSrc={
+          loggedInUser?.username === profile.slug
```

Comment:
why are you checking loggedInUser here ?

Reply:
@Udit-takkar ,
This component `EventTypeListHeading` is used in the map of `eventTypeGroups`. To make sure to update the avatar image of loggedInUser only I have to use that condition.

Without the loggedInUser check:
```  
imageSrc={avatar?.avatar || `${WEBAPP_URL}/${profile.slug}/avatar.png` || undefined} 
```
<img width="1252" alt="Screenshot 2023-05-02 at 3 38 53 PM" src="https://user-images.githubusercontent.com/12368271/235768312-ef8c696b-2f73-496a-b9b4-42a2a65eddd1.png">

With the loggedInUser check:
``` 
imageSrc={loggedInUser?.username === profile.slug ? avatar?.avatar : `${WEBAPP_URL}/${profile.slug}/avatar.png` || undefined}
```
        
<img width="1246" alt="Screenshot 2023-05-02 at 3 44 15 PM" src="https://user-images.githubusercontent.com/12368271/235769393-72d05b7a-682f-4cd0-aca8-5fb820f85a91.png">

Once you approve this change I have to update the same in the `MobileTeamsTab` function of the same file for Mobile view.

Reply:
@Udit-takkar ,
Let me know your thoughts on the above changes. I will revert if you think we should find a better way to handle this so that the rest of the code you can merge.

Reply:
Hi @Udit-takkar ,
Raised a separate PR(https://github.com/calcom/cal.com/pull/8694)  for the #8448 fixes. 
You can merge these changes.